handbook/data-stores/mongodb.md

```diff
@@ -0,0 +1,133 @@
+
+To successfully connect to a MongoDB replica set and create redundancy, you must specify a connection string in **NI SystemLink Server Configuration**.
+
+1. Log into the desktop of the SystemLink app server with administrator privileges
+1. Open **NI SystemLink Server Configuration**.
+1. Navigate to **NoSqlDataBase**.
+1. Click the **Connect to an externally managed NoSqlDatabase server** radio button.
+
+    !!!note "Secure remote connections"
+        NI recommends securing communications between your SystemLink server and MongoDB instance with TLS. For details, refer to [TLS for Remote MongoDB Instances](/network-security/network-security/#tls-for-remote-mongodb-instances).
+
+1. Select the **Use a custom connection string** checkbox.
+1. Enter your connection string. Refer to [Connection String URI Format](https://docs.mongodb.com/manual/reference/connection-string/) for details on structuring your connection string.
+1. Click **Test Connection** to ensure SystemLink can connect to the MongoDB instance.
+1. If the connection test was successful, click **Apply** to restart SystemLink service manager and connect to the MongoDB instance.
+
+### Connecting to MongoDB Atlas
+
+MongoDB Atlas provides simpler setup and adminstration compared to self hosted replica sets and stand alone deployments. MongoDB Atlas is a fully managed PaaS from MongoDB.
+
+If you have not setup an Atlas cluster before, refer to [Getting started with Atlas](https://docs.atlas.mongodb.com/getting-started/). Refer to [Setup Atlas Connectivity](https://docs.mongodb.com/guides/cloud/connectionstring/) for steps to obtain a connection string.
+
+!!! note
+    In the sample connection string provided by Atlas you will need to replace the instance of `myFirstDatabase` with `admin`. For example:
+
+    ```url
+    mongodb+srv://<username>:<password>@<cluster>/myFirstDatabase?retryWrites=true&w=majority
+    ```
+    
+    This will need to be updated to 
+    
+    ```url
+    mongodb+srv://<username>:<password>@<cluster>/admin?retryWrites=true&w=majority
+    ```
+
+!!!important "MongoDB Atlas free tier is unsupported by SystemLink"
+    Due to memory constraints on the Atlas free tier, you must use a paid tier for SystemLink to successfully connect. SystemLink supports all paid tiers of Atlas. If you need assistance evaluating Atlas with SystemLink please contact NI ([customer.requests@ni.com](mailto:customer.requests@ni.com)) or your local account manager.
+
+Because Atlas uses replica sets by default, your SystemLink server could be affected by the bug in the 3rd party MongoDB driver used by SystemLink as described in [Multi Node Deployments](#multi-node-deployments). If you cannot use the `mongodb+srv://` URI format, use the connection string generated for the Node.js driver version 2.0.14. This provided connection string is fully supported by SystemLink.
```

Comment:
```suggestion
Because Atlas uses replica sets by default, your SystemLink server could be affected by the bug in the third-party MongoDB driver used by SystemLink as described in [Multi Node Deployments](#multi-node-deployments). If you cannot use the `mongodb+srv://` URI format, use the connection string generated for the Node.js driver version 2.0.14. This provided connection string is fully supported by SystemLink.
```

Applies to everywhere this note is used in this doc